### PR TITLE
Calendar documentation

### LINKED ISSUE #127: Documentation for Calendar and Contacts
We should have an documentation for Calendar and Contacts. Therefore, we need some stuff. Feel free to add something 😁
- [x] Contacts
  - [x] Overview (#126)
  - [x] Add / Modify / Delete and Share Adressbooks 
  - [x] Add / Modify and Delete Contacts (Pictures, Adress, Groups, ...)
  - [x] CardDAV Backend Connect (Clients)
- [ ] Calendar
  - [ ] Overview
  - [ ] Add / Modify / Delete and Share Calendars 
  - [ ] Add / Modify / Delete Events (Name, Participants, Reminders, ...)
  - [x] CalDAV Backend Connect (Clients)

cc @hagengraf

### PR DESCRIPTION
**Pull request by MariusBluem**
Fixes #127 - to be improved later on. Please check if GIFs are supported by all browsers if we include it into our documentation 🤔

@georgehrke @MorrisJobke @rullzer @hagengraf @JonasDralle

### CODE REVIEW

File: user_manual/pim/calendar.rst [hunk 1]
```diff
@@ -1,3 +1,143 @@
...
+1. Click on the settings-icon labeled with `Settings & Import` at the left-bottom.
+
+2. After clicking on `+ Import Calendar` you can select a calendar file from your
+local device to upload.
+
+3. The upload can take some time and depends on how big the calendar you import
+is.
+
+.. note:: The Nextcloud Calendar application only supports iCalendar-compatible
+          `.ics`-files, defined in RFC 2445.
```
Comment (georgehrke) [inline user_manual/pim/calendar.rst]: RFC 2445 is what we don't support 😉 That's the legacy standard, version 1

We support RFC 5545.

File: user_manual/pim/calendar.rst [hunk 2]
```diff
@@ -1,3 +1,143 @@
...
+~~~~~~~~~~~~~~~~~
+
+If you want to transfer your calendar and their respective events to your Nextcloud
+instance, importing is the best way to do so.
+
+.. figure:: ./images/calendar_settings.png
+
+1. Click on the settings-icon labeled with `Settings & Import` at the left-bottom.
+
+2. After clicking on `+ Import Calendar` you can select a calendar file from your
```
Comment (georgehrke) [inline user_manual/pim/calendar.rst]: > you can select a calendar file

You can actually select multiple files :)
So maybe rephrase it to "you can select one or more calendar files"

File: user_manual/pim/calendar.rst [hunk 3]
```diff
@@ -1,3 +1,143 @@
...
+If you click on the red `Delete`-button the event will be removed from your
+calendar.
+
+Birthday calendar
+-----------------
+
+The birthday calendar is a auto-generated calendar which will automatically
+fetch the birthdays from your contacts.
+
+- You can not delete this calendar by your own. It will be re-generated after
```
Comment (georgehrke) [inline user_manual/pim/calendar.rst]: You actually can opt out of it starting with Nextcloud 15.

**Comment by georgehrke** [comment]
> Please check if GIFs are supported by all browsers if we include it into our documentation

Besides our fantastic new shiny docs.Nextcloud.com, we still generate PDF documentations and bundle them in the release. I don't think PDF viewers can display GIFs

**Comment by MariusBluem** [comment]
> Besides our fantastic new shiny docs.Nextcloud.com, we still generate PDF documentations and bundle them in the release. I don't think PDF viewers can display GIFs

At least after printing the PDF it will be lost xD

**Comment by MorrisJobke** [comment]
> Please check if GIFs are supported by all browsers if we include it into our documentation 🤔

Aren't they since the 90s? 🙈

File: user_manual/pim/calendar.rst [hunk 4]
```diff
@@ -1,3 +1,140 @@
...
+.. note:: The Nextcloud Calendar application only supports iCalendar-compatible
+          ``.ics``-files, defined in RFC 5545.
+
+Create a new Calendar
+~~~~~~~~~~~~~~~~~~~~~
+
+If you plan to setup a new calendar without transferring any old data from your
+previous calendar, creating a new calendar is the way you should go.
+
+.. figure:: ../images/calendar_create.png
```
Comment (MorrisJobke) [inline user_manual/pim/calendar.rst]: This image is missing.

File: user_manual/pim/calendar.rst [hunk 5]
```diff
@@ -1,3 +1,143 @@
...
+.. note:: The Nextcloud Calendar application only supports iCalendar-compatible
+          ``.ics``-files, defined in RFC 5545.
+
+Create a new Calendar
+~~~~~~~~~~~~~~~~~~~~~
+
+If you plan to setup a new calendar without transferring any old data from your
+previous calendar, creating a new calendar is the way you should go.
+
+.. figure:: images/calendar_create.png
```
Comment (MorrisJobke) [inline user_manual/pim/calendar.rst]: This is missing.

**Comment by MorrisJobke** [comment]
Beside the two missing images this PR looks good 👍

**Comment by MariusBluem** [comment]
Added another GIF which is also nice as static image :) @MorrisJobke @rullzer 
Please review & merge!

**Comment by MorrisJobke** [comment]
Somehow did the merge cause a revert of my commits ... let me redo this.

**Comment by MorrisJobke** [comment]
A rebase fixed the issue 👍

**Comment by MorrisJobke** [comment]
The usage of gifs breaks the PDF build.
